src/main/controller/UserController.ts

```diff
@@ -162,6 +163,15 @@ export class UserController {
         const userRepository = transactionalEntityManager.getCustomRepository(
           UserRepository
         );
+
+        const authenticatedUser = await userRepository.findByFirebaseId(
+          res.locals.firebaseUserId
+        );
+
+        if (authenticatedUser?.id !== id) {
+          throw new ForbiddenError();
+        }
```

Comment:
We should be able to do this now with my other suggestion!
```suggestion
        if (authenticatedUser.id !== id) {
          throw new ForbiddenError();
        }
```